doc/release-notes.md

```diff
@@ -31,15 +31,190 @@ Downgrading to a version older than v19.2.0 is not supported due to changes
 in the evodb database. If you need to use an older version, you must either
 reindex or re-sync the whole chain.
 
-# Notable changes
+# Release Notes
 
-- Core now categorizes asset unlock transactions as "Platform Transfers" on the Transactions tab in Dash-Qt and in the output of the `gettransaction` RPC (#6131)
-- Persist Coinjoin Denoms options changes made via GUI over restarts (#6208)
-- Fix incorrect payment predictions for evonodes in Dash-Qt and in RPC `masternode winners` (#6222)
-- `creditOutputs` entries in various RPCs that output transaction JSON are shown as objects now instead of being shown as strings (#6229)
-- Updated PGP key for builder 'pasta' to reflect new subkeys. You may need to reimport this key to validate signatures. (#6290)
-- Build failures on Ubuntu 24.10 / clang 19.1.1 resolved (#6328)
-- RPC errors in `masternode payments`, `getblock`, `getblockstats` related to Asset Unlock parsing have been fixed (#6336)
+## Notable Changes
+
+- **Asset Unlock Transactions (Platform Transfer)**
+    - Introduces a new fork `withdrawals` that changes consensus rules.
+    - **Validation Logic Update:**
+        - Allows using all **24 active quorums** and the most recent inactive quorum.
+        - Previous versions may refuse withdrawals with `bad-assetunlock-not-active-quorum` even if the quorum is active.
+    - **Withdrawal Limits Increased:**
+        - Flat **2000 Dash** per **576 latest blocks**.
+- **Increased Minimum Protocol Version**
+    - The minimum protocol version has been increased to **70216**.
+    - Masternode minimum protocol version has increased to **70235**.
+
+## Compatibility
+
+- **Dark Mode Appearance**
+    - Dash Core changes appearance when macOS "dark mode" is activated.
+
+- **glibc Requirement**
+    - The minimum required glibc to run Dash Core is now **2.31**. This means that **RHEL 8** and **Ubuntu 18.04 (Bionic)** are no longer supported.
+
+
+## Updated RPCs
+
+- **`getpeerinfo` Changes**
+    - `getpeerinfo` no longer returns the following fields: `addnode` and `whitelisted`, which were previously deprecated in v21.
+        - Instead of `addnode`, the `connection_type` field returns `manual`.
+        - Instead of `whitelisted`, the `permissions` field indicates if the peer has special privileges.
+    - *(#20755)*
```

Comment:
If including PR number, it should probably be an actual link to the PR for anyone not looking at the releases notes on Github. Do we typically include PR numbers for RPC changes?